AK/String.cpp

```diff
@@ -309,6 +311,70 @@ ErrorOr<String> String::replace(StringView needle, StringView replacement, Repla
     return StringUtils::replace(*this, needle, replacement, replace_mode);
 }
 
+bool String::contains(String const& needle, CaseSensitivity case_sensitivity) const
+{
+    return StringUtils::contains(bytes_as_string_view(), needle.bytes_as_string_view(), case_sensitivity);
+}
+
+bool String::contains(StringView needle, CaseSensitivity case_sensitivity) const
+{
+    return StringUtils::contains(bytes_as_string_view(), needle, case_sensitivity);
+}
+
+bool String::starts_with_bytes(StringView string, CaseSensitivity case_sensitivity) const
+{
+    return StringUtils::starts_with(bytes_as_string_view(), string, case_sensitivity);
+}
+
+bool String::ends_with_bytes(StringView string, CaseSensitivity case_sensitivity) const
+{
+    return StringUtils::ends_with(bytes_as_string_view(), string, case_sensitivity);
+}
+
+bool String::starts_with(u32 codepoint) const
```

Comment:
Why do we need all of these single code point APIs? Hardly seems useful, but perhaps I'm missing something

Reply:
They were implemented in DeprecatedString, but are not used (yet). I will remove them for now.